doc/navigating.rst

```diff
@@ -133,6 +133,44 @@ appear as a new tab in the settings dialog, where you can adjust its settings
 at 
 any time. Use **Account** > **Remove** to delete accounts. 
 
+File Manager Overlay Icons
+--------------------------
+
+The ownCloud sync client provides overlay icons, in addition to the normal file 
+type icons, for your system file manager (Explorer on Windows, Finder on Mac and 
+Nautilus on Linux) to indicate the sync status of your ownCloud files.
+
+The overlay icons are similar to the systray icons introduced above. They 
+behave differently on files and directories according to sync status 
+and errors. 
+
+The overlay icon of an individual file indicates its current sync state. If the
+file is in sync with the server version, it displays a green checkmark.
+
+If the file is ignored from syncing, for example because it is on your 
+exclude list, or because it is a symbolic link, it displays a warning icon.
+
+If there is a sync error, or the file is blacklisted, it displays an 
+eye-catching red X.
+
+If the file is waiting to be synced, or is currently syncing, the overlay 
+icon displays a blue cycling icon.
+
+When the client is offline, the icons remain in the state of the 
+last successful sync, as long as they are not touched locally. Once a file
+is edited locally, the icon will change to the 'waiting to sync' icon.
+
+The overlay icon of a synced directory indicates the status of the files in the 
+directory. If there are any sync errors, the directory is marked with a warning 
+icon.
+
+If the files inside the directory include ignored files that are marked with 
+warning icons, the parent directory remains in green mode.
```

Comment:
There will be a warning sign in case of error, even if there is ignored files.  The current wording seems to indicate that if there are ignored files, then the parent directory will always be green (even if there are other errors.)

Maybe reword it like : "include ignored files that are marked with warning icons does not change the status of the parent directies"